app/frontend/src/features/messages/Messages.tsx

```diff
@@ -44,7 +53,7 @@ export function HostRequestsSentNotification() {
 
 const labels = {
   all: "All",
-  groupchats: <MessagesNotification />,
+  chats: <MessagesNotification />,
   hosting: <HostRequestsReceivedNotification />,
```

Comment:
I also considered something like `[hostingRequestsRoute.split("/")[hostingRequestsRoute.length - 1]]: 
 <HostRequestsReceivedNotification />,` but seems way to ugly. But it would mean it doesn't need manual syncing.

Reply:
`[hostingRequestsRoute.split("/").pop()]` ?

Reply:
Hmm not sure about that, seems a little over abstracted for me. If the label does update, I suspect we'd want to rename the component name on the right hand side too, so it wouldn't be too much work to update both in one go? I doubt this will change enough to be a pain?

Reply:
agreed